package.json

```diff
@@ -145,7 +145,11 @@
       "output": "build/native"
     },
     "mac": {
-      "target": "dmg",
+      "target": {
+        "target": "dmg",
+        "arch": "universal"
+      },
+      "mergeASARs": false,
```

Comment:
This fix came from: https://github.com/electron-userland/electron-builder/issues/6735#issuecomment-1075372551